documentation/2.0/content/userguide/tools/create.md

```diff
@@ -58,3 +58,23 @@ topology:
     AdminServerName: "admin-server"
     ProductionModeEnabled: false
 ```
+## Parameter table for createDomain
+| Parameter | Definition | Default |
+| --- | --- | --- |
+| -archive_file | The path to the archive file to use.  If the -model_file argument is not specified, the model file in this archive will be used.  This can also be specified as a comma-separated list of archive files.  The overlapping contents in each archive take precedence over previous archives in the list. |    |
+| -domain_home | Required if -domain_parent is not used. The full directory and name where the domain should be created.
+| -domain_parent | Required if -domain_home is not used. The parent directory where the domain should be created. The name is the domain name in the model. |    |
+| -domain_type | The type of domain (e.g., WLS, JRF).  This controls the domain templates and template resource targeting. | WLS |
+| -java_home | The Java Home to use for the new domain. If not specified, it defaults to the value of the JAVA_HOME environment variable. |    |
+| -model_file | The location of the model file to use.  This can also be specified as a comma-separated list of model locations, where each successive model layers on top of the previous ones. |    |
+| -oracle_home | The existing Oracle Home directory for the domain. This is required unless the ORACLE_HOME environment variable is set. |    |
```

Comment:
All -oracle_home entries should be the same.  Pending the change above, copy it here.